packages/core-sdk/src/client.ts

```diff
@@ -32,6 +33,8 @@ export class StoryClient {
   private _dispute: DisputeClient | null = null;
   private _ipAccount: IPAccountClient | null = null;
   private _royalty: RoyaltyClient | null = null;
+  private _spg: SPGClient | null = null;
```

Comment:
I think we can rename this to NftClient to make it more user friendly. It can host all nft collection related functions that ties to SPG

Reply:
I see your point, but I think we should keep it as SPG since it might also deal with other functionalities in the future. Currently, it's a wrapper for core functions and an NFT collection manager. 

I also think it's better to move all SPG-related functions into SPG since they're an abstraction layer above the core calls that existing resources make (e.g. IPAsset resource calls core functions vs. SPG resource calls SPG that calls core functions).

Reply:
Following the discussion, we are switching NftClient to maintain the data type -> resource relationship. Resolved 24b51df